examples/standalone/ticksleep/src/main.c

```diff
@@ -0,0 +1,16 @@
+#include <tice.h>
+#include <time.h>
+
+int main(void)
+{
+    /* Clear the homescreen */
+    os_ClrHome();
+
+    /* Wait for 5 seconds */
+    os_PutStrFull("Waiting...");
+    ticksleep(5 * CLOCKS_PER_SEC);
+    os_PutStrFull(" Done!");
+    delay(100);
```

Comment:
why is there a delay here?

Reply:
That's just for the autotest to test the timing. It gives a 100ms target to check for after the expected time has elapsed.